docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -156,31 +151,29 @@ ends with lines similar to these:
 
 .. Tip::
 
-  If you want to stop the validator, enter CTRL-c in the validator's terminal
-  window. A single CTRL-c starts a graceful shutdown. Enter multiple CTRL-c
-  characters to force the validator to stop quickly.
+  If you want to stop the validator, enter CTRL-c several times
+  in the validator's terminal window.
 
-  You can stop any other running Sawtooth component by entering
-  CTRL-c in the appropriate window.
+Other Start-up Options for the Validator
+----------------------------------------
 
-.. note::
+In the ``sawtooth-validator`` command above, the `-vv` flag sets the log level
+to include DEBUG messages. To run the validator with less logging output,
+use `-v` or omit the flag.
 
-  By default, the validator listens on the loopback interface for both network
-  and component communications. To change the interface and port used, the
-  `--bind` flag can be used. (See :doc:`/cli/validator` for more information
-  on validator flags.) The following command is equivalent to the default
-  behavior::
+By default, the validator listens on the loopback interface for both network
+and component communications. To change the interface and port used, the
+`--bind` flag can be used. See :doc:`/cli/validator` for more information
+on the available flags.  The following command is equivalent to the default
+behavior::
 
     sudo -u sawtooth sawtooth-validator -vv --bind network:tcp://127.0.0.1:8800 --bind component:tcp://127.0.0.1:4004
 
-
-.. note::
-
-  The validator can process transactions in serial (the default) or parallel
-  with no difference in the state produced. To process in parallel, use the
-  option ``--scheduler parallel``. To get the most benefit from the parallel
-  option, start multiple transaction processors for the types of transactions
-  where there is an expected high volume.
+The validator can process transactions in serial (the default) or parallel
+with no difference in the state produced. To process in parallel, use the
+option ``--scheduler parallel``. To get the most benefit from the parallel
+option, start multiple transaction processors for the types of transactions
+where there is an expected high volume.
```

Comment:
Should this change be in the previous commit?

Reply:
This isn't a technical change - I just moved some words around (put the default info in the first sentence).  I intended this commit to have the "rewrite" changes.  The other one is the "fix typos" commit.

Reply:
Following up to say that yes, this change SHOULD have been in the other PR, but wasn't (as you helpfully noted).  Thanks for this catch!  The original ubuntu.rst changes were already merged, and I'm closing this confusing PR, so I'll make this correction in a later PR.  Thanks again!